pandas/tests/io/parser/test_textreader.py

```diff
@@ -341,6 +341,17 @@ def test_empty_csv_input(self):
         df = read_csv(StringIO(), chunksize=20, header=None, names=["a", "b", "c"])
         assert isinstance(df, TextFileReader)
 
+    def test_blank_lines_between_header_and_data_rows(self):
```

Comment:
this test needs to be in test_common.py and use the all_parsers fixture so it testsed.

Reply:
done